recipes/tests.rb

```diff
@@ -489,6 +489,7 @@ module load intelmpi && mpirun --help | grep 'Version 2019 Update 7'
       raise "Expected number of nfs threads configured to be #{expected_threads} but is actually #{nfs_threads}"
     end
   end
+  notifies :restart, "service[#{node['nfs']['service']['server']}]", :before
```

Comment:
can't this hide potential bugs in case nfs server is not restarted by the config recipe?